### PR TITLE
VEGA-2256 Switch to new digital LPA subtypes #minor

### PR DESCRIPTION
**Pull request by townxelliot**
## Checklist

- [x] I have updated the end-to-end tests to work with my changes
- [ ] I have added styling for Dark Mode
- [ ] I have checked that my UI changes meet accessibility standards

### CODE REVIEW

File: web/template/layout/case-details.gohtml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {{ define "case-details" }}
-    <p class="govuk-body"><strong>{{ if eq .Case.SubType "pfa" }}PFA{{ else if eq .Case.SubType "hw" }}HW{{ end }} {{ .Case.UID }}</strong></p>
+    <p class="govuk-body"><strong>{{ if eq .Case.SubType "property-and-affairs" }}PA{{ else if eq .Case.SubType "personal-welfare" }}PW{{ end }} {{ .Case.UID }}</strong></p>
```
Comment (gregtyler) [inline web/template/layout/case-details.gohtml:2]: This is used for legacy cases too, so still needs to support HW and PFA

File: web/template/payments.gohtml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-s govuk-!-padding-bottom-2">
-                <strong>{{ if eq .Case.SubType "pfa" }}PFA{{ else if eq .Case.SubType "hw" }}HW{{ end }} {{ .Case.UID }}</strong>
+                <strong>{{ if eq .Case.SubType "property-and-affairs" }}PA{{ else if eq .Case.SubType "personal-welfare" }}PW{{ end }} {{ .Case.UID }}</strong>
```
Comment (gregtyler) [inline web/template/payments.gohtml:9]: Similarly, this template is used for legacy LPAs too

**Review by gregtyler — APPROVED** [review]

**Comment by townxelliot** [comment]
The mix of "subtype" and "subType" is bothering me...

File: web/template/mlpa-documents.gohtml [hunk 3]
```diff
@@ -87,7 +87,7 @@
                         <td class="govuk-table__cell">
                             {{ if eq .Direction "Incoming" }}
                                 {{ .Type }}
-                                {{ if .SubType }}<br />&ndash; {{ .SubType }}{{ end }}
+                                {{ if .SubType }}<br />&ndash; {{ subtypeShortFormat .SubType }}{{ end }}
```
Comment (gregtyler) [inline web/template/mlpa-documents.gohtml:90]: This is a document subtype, which is already a human-readable string (e.g. "Reduced fee request evidence") so doesn't need to be translated
```suggestion
                                {{ if .SubType }}<br />&ndash; {{ .SubType }}{{ end }}
```

Comment (townxelliot) [reply web/template/mlpa-documents.gohtml]: I feel like this should have some sort of test around it.

**Review by gregtyler — DISMISSED** [review]

**Review by gregtyler — APPROVED** [review]

**Pull request merged**